### PR TITLE
Improve README and CONTRIBUTING guides

### PR DESCRIPTION
**Pull request by stefannibrasil**
The README now contains steps to generating the code coverage report, how to use the SkunkScore as a refactoring guide, and a separate Contributing guide. I also cleaned up a bit the skunk report to help people focus on what matters. It's my first time opening a Pull Request here, curious to see your thoughts!

I have a couple of questions of things I'd like to add in the README:
- [ ]  What is a healthy SkunkScore to add as an example for people to compare their current score with?
- [x]  Are there any Changelog good practices/guidelines to add in the Contributing guide?

Thanks!

I will abide by the [code of conduct](https://github.com/fastruby/skunk/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+To install this gem onto your local machine, run:
+
+`bundle exec rake install`.
+
+## Architecture
+
+This project follows the typical structure for a gem: code is located in `lib` and tests are in `test`.
+
+Under `/lib/skunk`, you find the CLI commands. Under `lib/skunk/rubycritic` you find skunk's extension from `rubycritic` gem. These are likely the files you'll be most interested in.
```
Comment (bronzdoc) [inline CONTRIBUTING.md:34]: This looks good, I just think this: `These are likely the files you'll be most interested in.` is not necesary

File: README.md [hunk 2]
```diff
@@ -4,238 +4,186 @@
...
-
-The main goal of the SkunkScore is to serve as a compass in your next
-refactoring adventure. It will help you answer these questions:
+The main goal of the SkunkScore is to serve as a compass in your next refactoring adventure. It will help you answer these questions:
 
 - What can I do to pay off technical debt?
 - What are the most complicated files with the least code coverage?
-- What are good candidates for your next test-writing efforts?
-- What are good candidates for your next refactoring efforts?
+- What are good candidates for my next test-writing efforts?
```
Comment (bronzdoc) [inline README.md:27]: why this change?

**Review by bronzdoc — COMMENTED** [review]
This looks like a good improvement, thanks so much for this @stefannibrasil!

Just left a few comments

File: CONTRIBUTING.md (hunk 1 above)
Comment (stefannibrasil) [reply CONTRIBUTING.md:34]: thanks @bronzdoc I agree.

File: README.md (hunk 2 above)
Comment (stefannibrasil) [reply README.md:27]: I think it would be nice to have all the questions in the first person: "my" and "I" :eyes:

**Comment by kindoflew** [comment]
hi @stefannibrasil! thanks for contributing!

I can answer at least one of your questions -- relating to `CHANGELOG.md` stuff. 

We use a format based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0/). The important things to note are that new additions get added under the `main (unreleased)` heading and should be a link to the PR with the following format:

```
* [<FEATURE | BUGFIX | CHORE>: Description of changes](github.com/link/to/pr)
```

As far as a good Skunk Score, I'm not really sure, but perhaps @etagwerker could help answer that one?

**Comment by stefannibrasil** [comment]
@kindoflew @bronzdoc thank you for your review! I added more details about the Changelog.

@etagwerker do you have any thoughts on adding a "healthy" SkunkScore to add as an example for people to compare their current score with?

**Comment by kindoflew** [comment]
ping @etagwerker :shipit:

**Comment by bronzdoc** [comment]
We really appreciate your contribution @stefannibrasil! I apologize for the delay, I think this is good to go and we can improve it later when we tackle https://github.com/fastruby/skunk/issues/95

**Review by bronzdoc — APPROVED** [review]
